src/site/content/en/blog/native-file-system/index.md

```diff
@@ -246,27 +286,40 @@ butDir.addEventListener('click', async (e) => {
 });
 ```
 
+## What's new/changed? {: #whats-new }
+
+Chrome 82
+
+* Support for [writable streams](#save-to-disk) was added, and the previous
+  method for writing to disk (`FileSystemWriter`) was deprecated.
+* File handles can now be serialized and stored in IndexedDB, or sent via `postMessage`
+  to other windows or workers within the same origin. Note that permissions
+  are not retained between browser sessions. For example, when a browser tab
+  is re-opened, and a file handle is obtained from IndexedDB, the user will
+  need to grant permission to read/write to the file again.
+* Added support for `isSameEntry()`, which returns true if two entries
+  represent the same file or directory.
+* Updated usage indicators to indicate whether the user has granted the domain
+  permission to files, including a new read-only indicator.
+
 ## What's currently supported? {: #whats-supported }
 
 We're still working on some of the implementation for the
 Native File System API, and not everything in the [spec][spec]
 (or [explainer][explainer]) has been completed.
 
-As of Chrome 78, the following functionality is not available, or
+As of Chrome 82, the following functionality is not available, or
 doesn't match the spec:
 
-* Handles are not serializable, meaning they cannot be passed via
-  `postMessage()`, or stored in IndexedDB.
-* Non-atomic writes (i.e. calls to `FileSystemFileHandle.createWriter()`
+* Non-atomic writes (i.e. calls to `FileSystemFileHandle.createWritable()`
```

Comment:
I'm not sure if we should even still mention this, or if we do we should definitely rephrase it. There is no mention of "inPlace" anymore in the spec (other than a note mentioning that there isn't currently any support for non-atomic writes).

We still don't really have a story/solution for use cases where we do need non-atomic writes though, so not sure what we should mention here.

Reply:
Removed this whole section for the time being.